src/fides/api/alembic/migrations/versions/14acee6f5459_translation_data.py

```diff
@@ -0,0 +1,651 @@
+        if us_config_id := US_MAPPING.get(component):
+            configs.append(reconciled_experience_config_map.get(us_config_id))
+        for config in configs:
+            if config:
+                config["accept_button_label"] = eec["accept_button_label"]
+                config["acknowledge_button_label"] = eec["acknowledge_button_label"]
+                config["banner_description"] = eec["banner_description"]
+                config["banner_enabled"] = eec["banner_enabled"]
+                config["banner_title"] = eec["banner_title"]
+                config["description"] = eec["description"]
+                config["privacy_policy_link_label"] = eec["privacy_policy_link_label"]
+                config["privacy_policy_url"] = eec["privacy_policy_url"]
+                config["privacy_preferences_link_label"] = eec[
+                    "privacy_preferences_link_label"
+                ]
+                config["reject_button_label"] = eec["reject_button_label"]
+                config["save_button_label"] = eec["save_button_label"]
+                config["title"] = eec["title"]
+                config["needs_migration"] = True
+
+    if has_existing_experience_configs:
+        return reconciled_experience_config_map
+
+    # if no existing experience configs are found, don't return the map, because no experience config migration is needed.
+    # instead, we'll rely on the OOB loading of experience configs on server startup.
+    # basically, this is allows us to bypass the data migration if we're not actually migrating a db that's been in use!
+    return {}
+
+
+def migrate_experiences(bind):
+    """
+    Migrates data from any existing ExperienceConfig records to the new, translation-based data model, and reconciles
+    the old ExperienceConfig records with new, OOB ExperienceConfig records.
+
+    - First the necessary existing ExperienceConfig and PrivacyNotice data is retrieved and stored in memory
+    - It is then, in memory, reconciled with our new OOB ExperienceConfig records, which are loaded from a YAML definition on disk
+    - Then, we remove all existing Experience and ExperienceConfig records to start from a 'blank slate'
+    - Then, based on the reconciled existing ExperienceConfig data, we create new ExperienceConfig and Experience records
+    """
+
+    def create_new_experience_config_templates(experience_configs):
+
+        for experience_config in experience_configs:
+            # for config template record:
+            # - `id` is the `origin` field from the config template record loaded from disk
+            create_experience_config_template_query = text(
+                """
+                INSERT INTO experienceconfigtemplate (id, component, name, disabled, allow_language_selection, dismissable, privacy_notice_keys, regions)
+                VALUES (:id, :component, :name, :disabled, :allow_language_selection, :dismissable, :privacy_notice_keys, :regions)
```

Comment:
I need to get clear on what allow_language_selection and auto_detect_language should be out of the box on the templates.  I was told "true" for both. They are true at the db level and allow_language_selection is false in the template.  I think they should both be false in the template, because whether these values can be true is also a factor of whether they've paid for translations or the ability to add translations right?

Reply:
OK updates from in-person meeting:

`allow_language_selection` is now false in the db (was already false in the template).  `Auto_detect_language` has been given a true value in the template here https://github.com/ethyca/fidesplus/pull/1334.

So we might add auto_detect_language to the regenerated templates we use here, and add that field to this query?